README.md

```diff
@@ -154,7 +157,6 @@ Each Impact Project may appoint 2 representatives to the CPC as outlined in the
 According to the [CPC Charter](./CPC-CHARTER.md#voting-members), each of the At Large and Incubating Projects may nominate a candidate to fill one of two voting seats on the CPC which represent this group of projects as a whole. From those nominees, two voting members are chosen via an [election process outlined in the CPC Governance](./governance/GOVERNANCE.md#election-process-for-non-impact-voting-cpc-members).
 
 - Jordan Harband ([@ljharb](https://github.com/ljharb), HeroDevs)
-- Ulises Gascón ([@UlisesGascon](https://github.com/ulisesgascon), NodeSource)
```

Comment:
does this mean we need a new election for another at large project rep?

Reply:
I think it remains vacant, esp because there was no election for this; only you and Ulises had volunteered.